_posts/2025-06-25-mast-heatmap.md

```diff
@@ -0,0 +1,117 @@
+
+#### Projection 1: Ecliptic
+As a quick refresher, CAOM uses coordinates aligned to the celestial pole, which is aligned to our rotating Earth. A declination of +90 is directly overhead the north pole, while -90 is directly overhead the south pole. Our first transformation is to the ecliptic plane, aligned instead to Earth's orbit around the sun:
+![a diagram showing how the ecliptic plane is defined](assets/img/posts/20250625/Earths_orbit_and_ecliptic.svg)
+<small>By <a href="//commons.wikimedia.org/wiki/User:CielProfond" title="User:CielProfond">CielProfond</a> - <span class="int-own-work" lang="en">Own work</span>, <a href="https://creativecommons.org/licenses/by-sa/4.0" title="Creative Commons Attribution-Share Alike 4.0">CC BY-SA 4.0</a>, <a href="https://commons.wikimedia.org/w/index.php?curid=156276169">Source</a></small>
+
+Ecliptic coordinates are useful for space-based telescopes, since the sun is — by definition —always at a declination of 0º. In fact, the [survey design of the TESS spacecraft](https://youtu.be/Q4KjvPIbgMI&t=115) takes advantage of this by always keeping one camera on the ecliptic pole, with the others facing opposite the sun.
+
+![the MAST heatmap, reprojected to the plane of the ecliptic](assets/img/posts/20250625/mast_wallpaper_ecl_blue.png)
+<small>Ohhhh, so that's why those observations make a straight line around the plane of the ecliptic! They're from TESS.</small>
+#### Projection 2: Galactic
+Galactic coordinates are, somewhat counter-intuitively, still centered on our Sun. In this case, 0º longitude is defined to be the line between the sun and the center of the galaxy. Latitude measures the angle of an object relative to the flat disk of the galaxy, as seen from Earth.
+
+![](assets/img/posts/20250625/mw.jpg)
+<small>By NASA/JPL-Caltech/ESO/R. Hurt - <a rel="nofollow" class="external free" href="http://www.eso.org/public/images/eso1339e/">http://www.eso.org/public/images/eso1339e/</a>, Public Domain, <a href="https://commons.wikimedia.org/w/index.php?curid=28274906">Source</a></small>
+
+The nice thing about using the galactic coordinate system is that most stars are near the plane of the galaxy, so it shows up quite nicely on our plot!
+![the mast heatmap, projected to galactic coordinates](assets/img/posts/20250625/mast_wallpaper_gal_blue.png)
+<small>The Milky Way, as seen by MAST. Come on, TESS! Observe that last little part in the middle so we can see the whole galaxy.</small>
+## Conclusions and Other Thoughts
+It's pretty dang cool that through metadata alone, we can reconstruct the shape and even texture of the milky way. These images, which are gorgeous, are also a powerful reminder that MAST has a lot of data. And this is a reminder that those data are free for anyone to access, so check them out through our [search forms](https://mast.stsci.edu/search/ui) or our [astroquery API](https://spacetelescope.github.io/mast_notebooks/intro.html)!
```

Comment:
Do we have a link for TAP documentation? I find it interesting to list the search form and astroquery, neither of which was used to fetch this data. It might still be possible to use them, but TAP is more direct.